RedHatInsights/patchman-ui
# fix(ESSNTL-4494): Fix group column and filter

This removes the custom implementation of group filter and makes sure the column and filter are working according to requirements.

## How to test

Try to do basic filtering with groups, verify the group column: 'N/A' for hosts without any group and a group name for hosts that belong to any of account's groups.

## Screenshots

<img width="1271" alt="Screenshot 2023-08-31 at 22 59 13" src="https://github.com/RedHatInsights/patchman-ui/assets/31385370/59563a86-51ab-4940-839e-b5b1db40bb1a">
